assets/superagent.js

```diff
@@ -682,7 +682,6 @@
  */
 
     function Response(req, options) {
-      options = options || {}
```

Comment:
Are we sure this wasn't supposed to be used? Like `this.options`?

Reply:
It's not passed downstream and there's no reference to it anywhere else, so I think it should be safe.